netbox/netbox/views/generic/bulk_views.py

```diff
@@ -676,6 +676,9 @@ def post(self, request):
         else:
             form = self.form(initial={'pk': request.POST.getlist('pk')})
             selected_objects = self.queryset.filter(pk__in=form.initial['pk'])
+            for object in selected_objects:
+                # Do something to raise error message to user
+                pass
```

Comment:
The view should already handle the necessary validation. Did you run into something that isn't covered?

Reply:
I believe I was just trying to raise an error when selecting a device without a name. Going over things, I definitely prefer your previous suggestion of `obj.name or ''`. Removing these lines.